tag/nitag.yml

```diff
@@ -1503,6 +1452,15 @@ definitions:
     example:
       type: DOUBLE
       value: "3.14"
+  TagPathsAndCountValues:
```

Comment:
Ditto - "TagPathWithCountAggregate". In this case we probably want the title to be singular as well. Again, feel free to drop this if there's pressure to just get the codegen fixed quickly.